src/OrchardCore/Orchard.DisplayManagement.Liquid/Fluid/Tags/HelperTag.cs

```diff
@@ -117,7 +117,7 @@ public HelperStatement(ArgumentsExpression arguments, IList<Statement> statement
 
                 foreach (var attribute in descriptor.Attributes)
                 {
-                    if (attribute.IsNameMatch(attributeName))
+                    if (attribute.IsNameMatch(attributeName) || attribute.IsNameMatch("asp-" + attributeName))
```

Comment:
No, this is too hacky.
Instead we talked about having the resource tag helpers directly available as liquid tags. The liquid tag's WriteAsync will just call the tag helper 's WriteToAsync to reuse some code.